reportportal_client/service.py

```diff
@@ -267,6 +268,8 @@ def start_test_item(self,
         if parameters:
             parameters = _dict_to_payload(parameters)
 
+        code_ref = sys.argv[0]
```

Comment:
It must be an optional parameter passed by the agent. Taking the first value from the sys.argv does not guarantee that it will be a path to the test script/module.